### DISCUSSION TITLE
Can't pair Ikea Knycklan

### DISCUSSION THREAD

**Discussion by sliwma** [Koenkk/zigbee2mqtt#10627] [Devices]
Hey, I recently got two Knycklan valves from Swedish Ikea, and I have an issue with both - I can't add pair them.
I can bind them (restart valve + hold restart on remote for few seconds), I can pair the remotes but not the valves.

Anyone had such an issue? Are there any prerequisites that needs to be complied with, like specific channel?

**Comment by MarkoMarjamaa** [comment]
Had no problem. Earlier had Raspbee II + ZHA, now zzh! + zigbee2mqtt. Transfer was very smooth. 
" To reset the receiver, press the on and off buttons simultaneously for 5 seconds." 
You cannot pair the valve with remote _and_ zigbee-hub, only with one. 

Problem with zigbee2mqtt is, the valve works ok, but there is no sensor for water leak. I need also the water leak sensors, because I have one valve for two dish washers and I need two leak sensors. Then I have another valve for refrigerator, but its leak sensor is under dish washer and another manufacturers leak sensor under refrigerator.

**Reply by sliwma** [reply]
I figured out what's wrong - I was holding buttons not long enough, so it was resetting the valve, not the Zigbee connection - works now, thanks!
Pairing with remote and with z2m can be somehow achieved by binding.

+1 about the water leak status - was it available in ZHA? I'm using mine for washer and dishwasher and I have another sensors to have the leak status.

**Reply by MarkoMarjamaa** [reply]
Yeah, zha had separate binary sensor for water leak. 
I have also different valve for main water, so it would be nice the get the water leak info.

**Reply by sliwma** [reply]
I created an [issue](https://github.com/Koenkk/zigbee-herdsman-converters/issues/3949) regarding the missing water leak information.

**Reply by MarkoMarjamaa** [reply]
> I created an [issue](https://github.com/Koenkk/zigbee-herdsman-converters/issues/3949) regarding the missing water leak information.

I created custom converter to get the water leak info. Code is in the other thread.
